src/shell.zig

```diff
@@ -388,7 +388,7 @@ pub fn exec_options(
     }
 }
 
-/// Returns `true` if the command executed successfully with a zero exit code and an empty stderr.
+/// Returns `true` if the command executed successfully with a zero exit code.
```

Comment:
This is the second time the stderr check gets in the way (as benchmark logs to stderr). Before that, I get a false error somewhere where I was running `tool --version`, and was getting print out to stderr about the tool being installed transparently by some sort of version manager. So I figured it's better to keep it simple and just not check for stderr.